### DISCUSSION TITLE
scripts accessing custom variables from configuration.py

### DISCUSSION THREAD

**Discussion by jscooksey** [netbox-community/netbox#10994] [Q&A]
No doubt Im going to kick myself:

If I add variables in to configuration.py then how can I pull that data for use in a customer script or plugin?

**Comment by adparis99** [comment]
Out of curiosity, what kind of variable are you defining in configuration.py that you can't define elsewhere in NetBox?

**Reply by jscooksey** [reply]
I'm accessing another database, so host, db, user, password.
I can throw in a JSON file in the folder and pull from that, but just wondering....

**Comment by bogdancordos** [comment]
You can use variables from configuration.py by importing netbox.settings in your custom script (`import netbox.settings`) and call them by `netbox.settings.VARIABLE_NAME`
